hack/cluster-up.sh

```diff
@@ -23,3 +23,9 @@ echo "Enable hotplug"
 #Add the feature gate to the resource of type Kubevirt.
 ./kubevirtci kubectl patch -n kubevirt kubevirt.kubevirt.io kubevirt  -p '{"spec":  { "configuration": { "developerConfiguration": { "featureGates": ["HotplugVolumes" ] }}}}' -o json --type merge
 
+# TODO: update cluster VMI's with insecure registry...
+# need to run pod per node, and apply change and restart
+# ./kubevirtci kubectl apply -f ./deploy/host-pod.yaml
+# ./kubevirtci kubectl-tenant exec host-pod -- /bin/sh -c "`cat ./deploy/containerd-config.sh`"
```

Comment:
This is actually a pretty commonly used pattern. Even the openshift conformance tests do this sort of thing.

start a pod somewhere, then exec into it to do something that the tests need. :+1:

Reply:
just be aware that multiple nodes need this config

Reply:
that is right, a comment above already states it, 
I did test it manually